src/content/en/fundamentals/media/manipulating/cheatsheet.md

```diff
@@ -0,0 +1,372 @@
+      input=myvide.mp4,stream=audio,output=enc_myaudio.m4a \
+      --enable_widevine_encryption \
+      --key_server_url "https://license.uat.widevine.com/cenc/getcontentkey/widevine_test" \
+      --content_id "fd385d9f9a14bb09" --signer "widevine_test" \
+      --aes_signing_key "1ae8ccd0e7985cc0b6203a55855a1034afc252980e970ca90e5202689f947ab9" \
+      --aes_signing_iv "d58ce954203b7c9a9a9d467f59839249"
+
+## All Together Now
+
+### DASH/webm with Shaka Packager
+
+Not all steps are possible with Shaka Packager.
+
+1. Convert the file type and codec.
+
+    For this command you can use either `liborbis` or `libopus` for the audio codec. 
+
+    ```
+    ffmpeg -i glocken.mov -c:v libvpx-vp9 -c:a libvorbis -b:v 8M -vf setsar=1:1 -f webm glocken.webm
+    ```
+
+2. Create a Clear Key encryption key.
+
+    You'll need to open the key file and manually remove all whitespace
+    including the final carriage return.
+
+    ```
+    openssl rand 16 > media.key
+    ```
+
+4. Demux the audio and video, encrypt the new files, and output an mpd file.
+
+    The `-key` and `-key_id` flags are copied from the `media.key` file.
+
+    ```packager \```<br/>
+    ```  input=myvideo.webm,stream=video,output=myvideo_video.webm \```<br/>
+    ```  input=myvideo.webm,stream=audio,output=myvideo_audio.webm \```<br/>
+    ```  --enable_fixed_key_encryption --enable_fixed_key_decryption \```<br/>
+    ```  -key 7bbef28644876a545a40caaaad9b3200 -key_id 7bbef28644876a545a40caaaad9b3200 \```<br/>
+    ```  --mpd_output myvideo_vod.mpd```
+
+### DASH/mp4 with Shaka Packager
+
+Not all steps are possible with Shaka Packager.
+
+1. Convert the file type, video codec and bitrate.
+
+    The default pixel format, yuv420p is used because one isn't supplied in the
+    command line. The app will give you an error message that it is deprecated.
+    I've chosen not to override the default because, though deprecated yuv420p
+    is the most widely supported.
+
+    ```
+    ffmpeg -i mymovie.mov -c:v libx264 -c:a aac -b:v 8M -strict -2 mymovie.mp4
+    ```
+     
+2. Create a Clear Key encryption key.
+
+    You'll need to open the key file and manually remove all whitespace
+    including the final carriage return.
+
+    ```
+    openssl rand 16 > media.key
+    ```
+
+3. Demux the audio and video, encrypt the new files, and output a media
+   presentation description (MPD).
+
+    The `-key` and `-key_id` flags are copied from the `media.key` file.
+
+    ```packager \```<br/>
+    ```  input=mymovie.mp4,stream=audio,output=myaudio.m4a \```<br/>
+    ```  input=mymovie.mp4,stream=video,output=myvideo.mp4 \```<br/>
+    ```  --enable_fixed_key_encryption --enable_fixed_key_decryption \```<br/>
+    ```  -key 7bbef28644876a545a40caaaad9b3200 -key_id 7bbef28644876a545a40caaaad9b3200 \```<br/>
+    ```  --mpd_output myvideo_vod.mpd```
+    
+### Widevine
+
+The two previous examples used Clear Key encryption. For widevine the final two
+steps are replaced with this.
```

Comment:
We need to consider linking (or adding, maybe?) some docs defining Clear Key vs Widevine etc.